pkg/codegen/python/python.go

```diff
@@ -105,6 +105,10 @@ func pyName(name string, legacy bool) string {
 
 	var result strings.Builder           // The components of the name, joined together with underscores.
 	var currentComponent strings.Builder // The characters composing the current component being built
+
+	result.Grow(len(name) + len(name)/5)
+	currentComponent.Grow(len(name) + len(name)/5)
```

Comment:
What's the significance of `len(name) + len(name)/5`? Could we add a comment for it?

Reply:
Yep--sorry for the omission. Late-night coding :P